index.html

```diff
@@ -48,7 +57,180 @@ <h1>I’m Nurbol<br>Glad to see<br> you!</h1>
+                        <h2 id="eight">Multi-Post<br>Stories</h2>
+                        <ul class="frame">
+                            <li id="client">CANOPY</li>
+                            <li id="dot"> </li>
+                            <li id="role">Back End Dev</li>
+                            <li id="dot"> </li>
+                            <li id="year">2015</li></ul>
+                    </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="bottom">
+                    <ul class="tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li></ul>
+                    <button class="action">See Project</button>
+                </div>
+            </div>
+        </div>
+
+<!--3 work card-->
+
+<div class="works-card">
+    <img src="worksimg/Snapshoot Portfolio@2x (2).png">
+    <div class="info-block">
+        <div class="top">
+            <h2 id="six">Tonic</h2>
+            <ul class="frame">
+                <li id="client">CANOPY</li>
+                <li id="dot"> </li>
+                <li id="role">Back End Dev</li>
+                <li id="dot"> </li>
+                <li id="year">2015</li></ul>
+        </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="bottom">
+            <ul class="tags">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li></ul>
+            <button class="action">See Project</button>
+        </div>
+    </div>
+</div>
+
+<!--4 work card-->
+
+    <div class="works-card">
+        <img src="worksimg/Snapshoot Portfolio@2x (3).png">
+        <div class="info-block">
+            <div class="top">
+            <h2 id="eight">Multi-Post<br>Stories</h2>
+            <ul class="frame">
+                <li id="client">CANOPY</li>
+                <li id="dot"> </li>
+                <li id="role">Back End Dev</li>
+                <li id="dot"> </li>
+                <li id="year">2015</li></ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="bottom">
+                <ul class="tags">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li></ul>
+                <button class="action">See Project</button>
+            </div>
+        </div>
+    </div>            
+</div>    
+    </section>
+
+<!-- ─── About Me Section ─── -->    
+
+    <section class="aboutme">
+        <div class="text">
+            <h1>About<br>Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] Please let's stick to design with the choice of maybe `color` or `font-weight` so that the design feels awesome and more user friendly - aesthetics. 

- _**This is what we have now:**_
![image](https://user-images.githubusercontent.com/99715640/212458639-f302b03b-a261-4322-a430-515f0c81a0ea.png)

- _**Figma Design**_:
![image](https://user-images.githubusercontent.com/99715640/212458690-d00f205b-0061-444c-8904-b147c6716a88.png)